iocore/aio/AIO.cc

```diff
@@ -399,7 +399,7 @@ aio_queue_req(AIOCallbackInternal *op, int fromAPI = 0)
 static inline int
 cache_op(AIOCallbackInternal *op)
 {
-  bool read = (op->aiocb.aio_lio_opcode == LIO_READ) ? 1 : 0;
+  bool read = (op->aiocb.aio_lio_opcode == LIO_READ) ? true : false;
```

Comment:
Dang, it could have done a better job here. E.g.
`bool read = op->aiocb.aio_lio_opcode == LIO_READ;`